src/toil/batchSystems/options.py

```diff
@@ -105,14 +106,21 @@ def add_all_batchsystem_options(parser: Union[ArgumentParser, _ArgumentGroup]) -
         help="Should auto-deployment of the user script be deactivated? If True, the user "
         "script/package should be present at the same location on all workers.  Default = False.",
     )
+    parser.add_argument(
+        "--maxJobs",
+        dest="max_jobs",
+        default=sys.maxsize, # This is *basically* unlimited and saves a lot of Optional[]
```

Comment:
```suggestion
        default=sys.maxsize, # This is *basically* unlimited and saves a lot of Optional[]
```

Sure, or use `0`?

Reply:
If we do 0 we need to special-case 0 when we check against the limit. Though it would make it easier for the user to say "unlimited".